js/client-teambuilder.js

```diff
@@ -3154,7 +3154,7 @@
 				if (baseFormat.substr(0, 8) === 'pokebank') baseFormat = baseFormat.substr(8);
 				if (this.curTeam && this.curTeam.format) {
 					if (baseFormat.substr(0, 10) === 'battlespot' && baseFormat.substr(0, 19) != 'battlespotspecial13' || baseFormat.substr(0, 3) === 'vgc') set.level = 50;
-					if (baseFormat.substr(0, 2) === 'lc' || baseFormat.substr(0, 5) === 'caplc') set.level = 5;
+					if (baseFormat.substr(0, 2) === 'lc' || baseFormat.substr(0, 5) === 'caplc' || baseFormat.substr(-2) === 'lc') set.level = 5;
```

Comment:
> Should this also be refactored to use String#startsWith/String#endsWith?

Probably? ` baseFormat.substr(-2) === 'lc'` makes `baseFormat.substr(0, 5) === 'caplc'` reduandant if you look at it.

Reply:
I assumed that check should be kept in case we introduce CAP LC UU or something.